docs/upgrade_to_tails_2x.rst

```diff
@@ -0,0 +1,138 @@
+Upgrade Tails from 1.x to 2.x
+=============================
+
+Newer versions of SecureDrop leverage improvements to the Tails OS since the introduction of Tails 2.0 (and subsequent versions). It is critical to upgrade all Tails USBs to the latest version before upgrading SecureDrop. For some admins, this might be complicated, since the upgrade from 1.x to 2.x must be done manually. Please follow this guide to updating the entire system, and special attention to only use the Secure Viewing Station computer so you benefit from its airgap while transfering sensitive data.
+
+.. note::
+	You will need:
+
+	#. A "master" Tails USB
+	#. Transfer device
+	#. Your existing SecureDrop Tails USB sticks (Admin/Journo/Secure Viewing Station)
+
+1. Prepare the "master" Tails USB
+-------------------------------------
+
+Because Tails 2.x is incompatible with older versions, you must create a new "master" Tails USB stick for subsequent installations and upgrades to the USB sticks already in-use by your organization.
+
+To create this brand-new "master" Tails, follow the same :doc:`directions for provisioning the first USB sticks <set_up_tails>`.
+
+You may now boot into this new Tails from your airgapped Secure Viewing Station to perform the next step. At the Tails Greeter screen, be sure to enable admin privileges.
+
+2. Prepare the Transfer Device
+------------------------------
+
+While it's recommended to use a fresh USB stick for any backup operation, to cut down on cost and/or waste, you may also repurpose older USB sticks to function as Transfer Devices.
+
+Open the Disks Utility by navigating to Applications ▸ Utilities ▸ Disks.
+
+Insert your Transfer Device into a USB port and select it from the left column. Brand new devices sometimes have pre-configured partitions, which you will need to remove. Select any block of partitioned data, and click the minus (``-``) button to remove any unwanted partitions.
+
+|Unwanted Bloatware Partition|
+
+Click the plus (``+``) button to register a new partition to cover the entire available space. This should automatically be filled out for you in the wizard.
+
+Be sure to do a full wipe of the existing data during this step, especially if you choose to repurpose an older USB stick. Select "Overwrite existing data with zeroes (Slow)" from the "Erase" options. This does mean that the reformatting step will take some extra time, but it is the best way to insure that previous data from older transfers, or manufacturer bloatware, is removed from the Transfer Device.
+
+Give your new partition a complex, diceware-generated passphrase. 
```

Comment:
Maybe we should just say "strong passphrase"? I'd rather use that throughout the docs, and then provide our own definition of "strong passphrase" in one canonical location (if you think that would be useful).